app/src/main/res/layout/item_menu_drawer.xml

```diff
@@ -57,6 +57,15 @@
                 android:layout_marginEnd="@dimen/marginStandardSmall"
                 tools:text="42" />
 
+            <ImageView
```

Comment:
You should add a contentDescription, probably something like "contains unread mails"